### DISCUSSION TITLE
Feature Request: customize date format of folders

### DISCUSSION THREAD

**Discussion by mh166** [sircharlo/meeting-media-manager#336] [Ideas] [answered]
Hi there,

thanks for this sooo useful application! It really cuts down a lot of the time for preparing the media for the meetings! 🎉 Maybe you can make this even more awesome for us. 😉

At the moment, all the folders containing media are named like `YYYY-MM-DD`. For easier access, we use a directory structure like so: `YYYY-MM\YYYY-MM-DD - DDDD`. So for example, tomorrow's meeting is located `2022-05\2022-05-18 - Wednesday` (or `... - Mittwoch` to be precise, as we're a german congregation).

While I won't insist on keeping that subfolder there (which possibly would require more complex changes), I'd love to see the ability to have a custom date format for the folders. Having the weekday at the end of the folder helps immensely in finding the correct folder quickly.

**Comment by sircharlo** [comment, marked as answer]
Very interesting request. I agree that the subfolder creation would maybe be a step too far; however, implementing custom folder names is quite possible. I'll mull over the best way of implementing this.

**Reply by sircharlo** [reply]
Initial implementation [here](https://github.com/sircharlo/meeting-media-manager/commit/5aa64bdf69ab18bd8769ca5016b109c080d4ff66)

**Reply by mh166** [reply]
From what I can tell, it looks very good. Thank you so much! 😊
